utils/cloneByDate.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
And for the python files that are not really use as standalone scripts (despite they were in the past), we remove the shebang.